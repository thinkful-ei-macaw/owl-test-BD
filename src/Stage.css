.icon {
    height: 25px;
    width: 25px;
    padding: 5px;
}

.h2 {
    color: white;
    width: 200px;
    height: 75px;
    font-size: 16px;
    background-color: rgb(29, 29, 29);
    border-radius: 5px;
    text-align: center;
    display: flex;
    flex-direction: column;
    justify-content: center;
}

hr {
    width: 150px;
    height: 3px;
    background-color: red;
    border: none;
}

.user {
    display: flex;
    flex-direction: column;
    align-items: center;
    width: 50%;
    height: 85%;
}

.user-avatar {
    border-radius: 10em;
    background-color: blue;
    margin-top: 200px;
}