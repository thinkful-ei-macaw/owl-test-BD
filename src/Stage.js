import React from 'react';
import './Stage.css';

function Stage(props) {
    
    if (props.onStage){
    return (
        <div className='user'>
            <h2 className="h2">{props.name}<hr></hr></h2>
            <div className='icons'>
            <img className='icon' src="https://assets.dryicons.com/uploads/icon/svg/12209/cc0cdcf8-828c-4c43-8361-3db24f11ada3.svg" alt="muted speaker"/><img className='icon' src="https://as2.ftcdn.net/jpg/01/23/80/31/500_F_123803188_7f2vK45t4fJQ1UwBgb7dSEU2Yj6XEkaB.jpg" alt="magnifying glass"/></div>
            <img src={props.avatar} alt='user' className="user-avatar"/>
        </div>
    )
    }
}

export default Stage;