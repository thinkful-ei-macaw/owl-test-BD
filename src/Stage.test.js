import React from 'react';
import ReactDOM from 'react-dom'
import Stage from './Stage'
import renderer from 'react-test-renderer'

describe('<Stage />', () => {
  it('renders without crashing', () => {
      const div = document.createElement('div');

      // Pass required props in the usual way
      ReactDOM.render(
          <Stage
              name="Joe"
              avatar="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
              onStage={true}
          />,
          div
      );

      ReactDOM.unmountComponentAtNode(div);
  });

  it('renders this UI as expected', () => {
    // Render the component, as JSON
    const tree = renderer.create(<Stage
      name="Joe"
      avatar="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
      onStage={true}
  />).toJSON();
    // Check whether it matches the previous snapshot
    // Stored in __snapshots__/App.test.js.snap
    expect(tree).toMatchSnapshot(); 
});
});