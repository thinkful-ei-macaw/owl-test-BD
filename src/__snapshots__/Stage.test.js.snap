// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Stage /> renders this UI as expected 1`] = `
<div
  className="user"
>
  <h2>
    Joe
  </h2>
  <hr />
  <img
    alt="magnifying glass"
    src="https://as2.ftcdn.net/jpg/01/23/80/31/500_F_123803188_7f2vK45t4fJQ1UwBgb7dSEU2Yj6XEkaB.jpg"
  />
  <img
    alt="muted speaker"
    src="https://assets.dryicons.com/uploads/icon/svg/12209/cc0cdcf8-828c-4c43-8361-3db24f11ada3.svg"
  />
  <img
    alt="user"
    className="user-avatar"
    src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
  />
</div>
`;
